crc-admin-helper.spec.in

```diff
@@ -75,6 +76,9 @@ go test ./...
 #gopkgfiles
 
 %changelog
+* Tue May 31 2022 Anjan Nath <kaludios@gmail.com>
+- install macOS arm64 builds
```

Comment:
This can be added below the ` * Tue May 17 2022 Christophe Fergeau <cfergeau@redhat.com> - 0.0.11-1` line

Reply:
made the change, is it because of the version change? as that version would also contain this change?